src/olympia/bandwagon/permissions.py

```diff
@@ -8,3 +8,14 @@ def has_permission(self, request, view):
 
     def has_object_permission(self, request, view, obj):
         return self.has_permission(request, view)
+
+
+class AllowCollectionContributor(BasePermission):
+    """Allow a contributor of a collection to do stuff.  Be careful how this
+    is used - """
```

Comment:
I think I meant to add a warning about actions here and unless you're careful you end up giving everyone access to create/list collections because those actions don't call `has_object_permission`.  It's less than ideal.

Reply:
Maybe be more explicit in the docstring? Something like `"""Allow a contributor of a collection to do stuff. Be careful where this used as it can allow creating / listing objects if used alone in a ViewSet that has those actions."""`

Reply:
https://github.com/mozilla/addons-server/pull/6420/commits/d4b4ee0f964c5c2361a5636050330f52fa14206d